### PR TITLE
Multiple Versions of Cypress

### PR DESCRIPTION
**Pull request by SagarGaniga**
Adds a capability to provide cypress_version.

### CODE REVIEW

File: bin/commands/runs.js [hunk 1]
```diff
@@ -67,10 +67,15 @@ module.exports = function run(args) {
               logger.warn(Constants.userMessages.NO_PARALLELS);
             }
 
+            if (bsConfig.cypress_version && bsConfig.cypress_version !== data.cypress_version) {
+              logger.warn(Constants.userMessages.CYPRESS_VERSION_CHANGED);
```
Comment (suryart) [inline bin/commands/runs.js:71]: let's check the message?

File: bin/helpers/utils.js [hunk 2]
```diff
@@ -374,3 +374,9 @@ exports.getNetworkErrorMessage = (dashboard_url) => {
                   + Constants.userMessages.CHECK_DASHBOARD_AT  + dashboard_url
   return chalk.red(message)
 }
+
+exports.versionChangedMessage = (preferredVersion, actualVersion) => {
```
Comment (karanshah-browserstack) [inline bin/helpers/utils.js:378]: Can we add Spec for this?

File: bin/commands/runs.js (hunk 1 above)
Comment (SagarGaniga) [reply bin/commands/runs.js]: Updated

File: bin/helpers/utils.js (hunk 2 above)
Comment (SagarGaniga) [reply bin/helpers/utils.js:378]: Done

**Review by karanshah-browserstack — APPROVED** [review]

**Comment by prateekbstack** [comment]
LGTM

**Pull request merged**